File: src/main/java/com/serdeliverance/cryptowallet/domain/Portfolio.java
package com.serdeliverance.cryptowallet.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Portfolio {
    private User user;
    private List<CurrencyTotal> currencies;
    private BigDecimal totalInUSD;
    private LocalDateTime date;
}

File: src/main/kotlin/com/serdeliverance/cryptowallet/v2/domain/Portfolio.kt
package com.serdeliverance.cryptowallet.v2.domain

import java.math.BigDecimal
import java.time.LocalDateTime

data class Portfolio(
    val user: User,
    val currencies: List<CurrencyTotal>,
    val totalInUSD: BigDecimal,
    val date: LocalDateTime
)
